sdk/Sdk.Generators/FunctionExecutor/FunctionExecutorGenerator.Emitter.cs

```diff
@@ -170,21 +170,22 @@ private static string GetMethodBody(IEnumerable<ExecutableFunction> functions, b
             private static string EmitFastPath(ExecutableFunction function)
             {
                 var sb = new StringBuilder();
-               
+
                 if (!function.IsStatic)
                 {
                     sb.Append($"""
-                                 var instanceType = types["{function.ParentFunctionClassName}"];
+                                                var instanceType = types["{function.ParentFunctionClassName}"];
```

Comment:
Were these additional indentations required? Does the existing tests for this generator still passes?

Reply:
yes they all pass. Its so that all the indenting aligns and the pragama is inserted in the correct place

Reply:
Thanks @cliedeman  I downloaded your fork and ran the tests, but the "FunctionsFromMultipleClasses" tests are failing due to a mismatch in line breaks between the generated and expected code. Does this test pass for you? What OS are you using? I tested it on Windows.

Reply:
@satvu Would you mind doing a sanity check on this?

Reply:
Tested again to make sure

<img width="590" alt="image" src="https://github.com/user-attachments/assets/0abaee17-b5e2-4641-9b0c-0bbe2e55aa10" />
I am running on Mac

Reply:
Can you trigger the CI so I can see the failures?

Reply:
@cliedeman See below build logs. https://azfunc.visualstudio.com/public/_build/results?buildId=203441&view=logs&j=c7302286-4a09-5f6a-8490-0246cd26c4be&t=d825abe2-482a-5bbe-207a-50d08abb03a5

Reply:
@cliedeman Did you get a chance to look at the build logs ^ ?

Reply:
@kshyju got around to this. Its the hardcoded `\n`

Can't use `Environment.Newline` due to 
<img width="910" alt="image" src="https://github.com/user-attachments/assets/2be39bd6-8e43-40a0-954f-b863a9f738ec" />